### PR TITLE
feat(ai-monitoring): Langchain integration docs

### PR DESCRIPTION
**Pull request by colin-sentry**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/python/integrations/langchain/index.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+```bash
+pip install --upgrade 'sentry-sdk' 'langchain-openai' 'langchain-core'
+```
+
+## Configure
+
+If you have the `langchain` package in your dependencies, the Langchain integration will be enabled automatically when you initialize the Sentry SDK.
+
+An additional dependency, `tiktoken`, is required if you want to calculate token usage for streaming chat responses.
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/langchain/index.mdx:21]: Can we add this to the code sample below?

**Review by vivianyentran — CHANGES_REQUESTED** [review]
Thanks for adding this! Added some suggestions

File: docs/platforms/python/integrations/langchain/index.mdx (hunk 1 above)
Comment (colin-sentry) [reply docs/platforms/python/integrations/langchain/index.mdx:21]: It just needs to be installed, no code change.

File: docs/platforms/python/integrations/langchain/index.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+- The Langchain integration will connect Sentry with Langchain and automatically monitor all LLM, tool, and function calls.
+
+- All exceptions in the execution of the chain are reported.
+
+- Sentry is configured not to consider LLM and tokenizer inputs/outputs as PII. If you want to include them, set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts despite `send_default_pii=True`, configure the integration with `include_prompts=False` like in the Options section.
+
+## Options
+
+By adding `LangchainIntegration` to your `sentry_sdk.init()` call explicitly, you can set options for `LangchainIntegration` to change its behavior:
```
[the remark this answers is not in the document]
Comment (colin-sentry) [reply docs/platforms/python/integrations/langchain/index.mdx:69]: It is the only option for now.

File: docs/platforms/python/integrations/langchain/index.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+It may take a couple of moments for the data to appear in [sentry.io](https://sentry.io).
+
+## Behavior
+
+- The Langchain integration will connect Sentry with Langchain and automatically monitor all LLM, tool, and function calls.
+
+- All exceptions in the execution of the chain are reported.
+
+- Sentry is configured not to consider LLM and tokenizer inputs/outputs as PII. If you want to include them, set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts despite `send_default_pii=True`, configure the integration with `include_prompts=False` like in the Options section.
```
[the remark this answers is not in the document]
Comment (colin-sentry) [reply docs/platforms/python/integrations/langchain/index.mdx:65]: reworded

File: docs/platforms/python/integrations/langchain/index.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+It may take a couple of moments for the data to appear in [sentry.io](https://sentry.io).
+
+## Behavior
+
+- The Langchain integration will connect Sentry with Langchain and automatically monitor all LLM, tool, and function calls.
+
+- All exceptions in the execution of the chain are reported.
+
+- Sentry by default considers LLM and tokenizer inputs/outputs as PII. If you want to include them, set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts and outputs despite `send_default_pii=True`, configure the integration with `include_prompts=False` as shown in the [Options section](#options) below.
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/langchain/index.mdx:65]:
```suggestion
- Sentry considers LLM and tokenizer inputs/outputs as PII and, by default, does not include PII data. If you want include the data, then set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts and outputs despite `send_default_pii=True`, configure the integration with `include_prompts=False` as shown in the [Options section](#options) below.
```
I think when we say "by default" here, we're actually talking about the default behavior of how we treat PII (it's not included/sent), and that's the default behavior we're changing (sending the PII). 

I think the word "include" is also vague. Would it be more accurate to talk about this as data scrubbing? e.g. "Sentry considers LLM and tokenizer inputs/outputs as PII and, by default, scrubs PII data. If you want to send the data unscrubbed, then set `send_default_pii=True` in the `sentry_sdk.init()` call."

**Review by vivianyentran — APPROVED** [review]
Left another suggestion but overall, looks good!

**Pull request merged**